test/specs/core.tooltip.tests.js

```diff
@@ -822,4 +822,63 @@ describe('Core.Tooltip', function() {
 		node.dispatchEvent(firstEvent);
 		expect(tooltip.update).not.toHaveBeenCalled();
 	});
+
+	describe('positioners', function() {
+		it('Should call custom positioner with correct parameters and scope', function() {
+
+			Chart.Tooltip.positioners.test = function(elements, eventPosition) {
+
+				expect(this instanceof Chart.Tooltip).toBe(true);
+				expect(elements instanceof Array).toBe(true);
+				expect(eventPosition.hasOwnProperty('x') && eventPosition.hasOwnProperty('y')).toBe(true);
```

Comment:
It doesn't work because if the method is not called, then expectations are not evaluated. We could use `spyOn` instead to be sure that the method is actually called, something like [that](/chartjs/Chart.js/blob/master/test/specs/core.plugin.tests.js#L81):

```javascript
var scope = null;

Chart.Tooltip.positioners.test = function() {
    scope = this;
    return {x: 0, y: 0};
};

spyOn(Chart.Tooltip.positioners, 'test');

// {...}

node.dispatchEvent(evt);

var fn = Chart.Tooltip.positioners.test;
expect(fn.calls.count()).toBe(1);
expect(fn.calls.first().args[0] instanceof Array).toBe(true);
expect(fn.calls.first().args[1].hasOwnProperty('x')).toBe(true);
expect(fn.calls.first().args[1].hasOwnProperty('y')).toBe(true);
expect(scope instanceof Chart.Tooltip).toBe(true);
```